app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -68,11 +66,9 @@ class ProfileListAdapter(
             binding.profileCard.setOnClickListener {
                 val args = Bundle()
                 args.putParcelable("Profile", item)
-                val navOptions = NavOptions.Builder().setEnterAnim(R.anim.nav_default_enter_anim).setExitAnim(
-                    R.anim.nav_default_exit_anim
-                ).setPopEnterAnim(R.anim.nav_default_pop_enter_anim).setPopExitAnim(R.anim.nav_default_pop_exit_anim)
-                    .build()
-                Navigation.findNavController(it).navigate(R.id.detailsFragment, args, navOptions)
+                val dialog = DetailsFragment()
```

Comment:
@coder2699 Please try to create the fragment instance using the sample code image shared on the Gitter channel as it is the recommended way of creating a fragment instance when bundles are also involved.